src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle {
+    private int length;
+    private int width;
+
+    public int getLength() {
+        return length;
+    }
+
+    public int getWidth() {
+        return width;
+    }
+
+    public void setLength(int length) {
+        this.length = length;
+    }
+
+    public void setWidth(int width) {
+        this.width = width;
+    }
+
+    public int getArea(){
+        return getLength() * getWidth();
+    }

```

Comment:
The Rectangle class should implement an interface to define its behavior. Currently, it only contains getters, setters, and a method to calculate the area. Consider creating an interface, such as 'Shape', with a method like 'getArea()', and have Rectangle implement this interface.